utils/woptipng.py

```diff
@@ -102,7 +95,10 @@ def verify_images(source_img, new_img, transform):
         os.rename(new_img, source_img) # move new image to old image // os.rename(src, dest)
     else: # we can't os.rename(image_after, image_before) because that would leave us with no source for the next transform
         shutil.copy(source_img, new_img) # override new image with old image // shutil.copy(src, dest)
-        debugprint(("TRANSFORMATION unsuccessfull: + " + transform + ", REVERTING " + source_img))
+        if not no_change:
```

Comment:
I would try to avoid the double negative and switch the if around or invert the `no_change` variable.

Reply:
Going to rename it `pixels_identical`.